ui/selectors/selectors.js

```diff
@@ -1423,23 +1388,31 @@ export const getAllConnectedAccounts = createDeepEqualSelector(
 );
 export const getConnectedSitesList = createDeepEqualSelector(
   getConnectedSubjectsForAllAddresses,
-  getMetaMaskIdentities,
+  getInternalAccounts,
   getAllConnectedAccounts,
-  (connectedSubjectsForAllAddresses, identities, connectedAddresses) => {
+  (connectedSubjectsForAllAddresses, internalAccounts, connectedAddresses) => {
     const sitesList = {};
     connectedAddresses.forEach((connectedAddress) => {
       connectedSubjectsForAllAddresses[connectedAddress].forEach((app) => {
         const siteKey = app.origin;
+
+        const lowerCaseConnectedAddress = connectedAddress.toLowerCase();
+
+        const internalAccount = internalAccounts.find(
```

Comment:
I thought we had an helper for this already? We kinda have the same pattern in another PR (but we were using `toChecksumAddress` IIRC, do you recall? Maybe this helper was not exported though...

Reply:
I think that was in mobile. We can probably just reuse `isEqualCaseInsensitive`